src/OrchardCore/OrchardCore.DisplayManagement/Html/CssOrchardHelper.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Generic;
+
+namespace OrchardCore;
+
+public static class CssOrchardHelper
+{
+    public static string GetLimitedWidthCssClasses(this IOrchardHelper _, params string[] additionalClasses)
+    {
+        return String.Join(' ', GetLimitedWidthCssClassList(additionalClasses));
+    }
+
+    public static string GetLabelCssClasses(this IOrchardHelper _, params string[] additionalClasses)
+    {
+        return String.Join(' ', GetLabelCssClassList(additionalClasses));
+    }
+
+    public static string GetStartCssClasses(this IOrchardHelper _, params string[] additionalClasses)
+    {
+        return String.Join(' ', GetStartCssClassList(additionalClasses));
+    }
+
+    public static string GetEndCssClasses(this IOrchardHelper _, params string[] additionalClasses)
+    {
+        return String.Join(' ', GetEndCssClassList(additionalClasses));
+    }
+
+    public static string GetEndCssClasses(this IOrchardHelper _, bool withOffset, params string[] additionalClasses)
+    {
+        return String.Join(' ', GetEndCssClassList(withOffset, additionalClasses));
+    }
+
+    public static string GetOffsetCssClasses(this IOrchardHelper _, params string[] additionalClasses)
+    {
+        return String.Join(' ', GetOffsetCssClassList(additionalClasses));
+    }
+
+    private static List<string> GetLabelCssClassList(params string[] additionalClasses)
+    {
+        return new List<string>(GetStartCssClassList(additionalClasses))
+        {
+            "col-form-label",
```

Comment:
This is something that we should not do. Hard coding Bootstrap classes in a helper class. I need to understand why it is required.

Reply:
@Skrypt 
It may sounds like a bad idea, but since we maintain TheAdmin theme, it makes things much easier to change down the line or even make it appsettings based so the user provide their own settings. Also, any view can use these helpers to adhere to the TheAdmin theme settings.

Reply:
@Skrypt I actually change the code so it driven from options. We can also make these options available per tenant by adding the settings to the Site. If people like this approach, I'll add UI settings to change the settings for each settings from the UI.